package/src/components/Channel/Channel.tsx

```diff
@@ -497,6 +497,7 @@ const ChannelWithContext = <
     handleReaction,
     handleRetry,
     handleThreadReply,
+    hasCameraPicker = takePhoto !== null,
```

Comment:
this is a pretty dangerous check `takePhoto !== null` as it relies on implementation details that may change in the future without us knowing about their side-effects. Can't we just export a new variable `isCameraPickerAvailable` (or something similar) instead?

Reply:
If this is a common pattern for all optional dependencies, I think we should make this change everywhere.

Reply:
I have pushed it, though not from the `native-package` dir, but it's handled in the `native.ts` file of the core that exposes functions and then within the code the functions are used to check the availability of the package.